dependencies.yaml

```diff
@@ -70,7 +70,7 @@ dependencies:
       - output_types: conda
         packages:
           - c-compiler
-          - cmake>=3.26.4
+          - &cmake_ver cmake>=3.26.4
```

Comment:
```suggestion
          - &cmake_ver cmake>=3.26.4,!=3.30.0
```

Reply:
blegh thank you (I just pushed this change as part of https://github.com/rapidsai/cugraph-pg/pull/23/commits/0dd89ed42e8ad2fcfa202c51f09d857ca298416e)